### PR TITLE
TASK: Persist intermediate node:repair results

### PR DESCRIPTION
**Pull request by robertlemke**
This change adds a couple of persistAll() calls to the different steps
in node:repair in order to save changes before the next steps begin.

That way, if one step fails, at least the previous steps have been
already persisted.

Additionally this may potentially prevent inconsistencies in cases where
the same node is created from NodeData multiple times without
acknowledging the Unit of Work.

Also contains a few smaller cosmetic changes to the console output.

### CODE REVIEW

File: TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Command/NodeCommandControllerPlugin.php [hunk 1]
```diff
@@ -297,15 +309,25 @@ protected function createChildNodesByNodeType(NodeType $nodeType, $workspaceName
...
                 if ($nodeCreationExceptions > 0) {
                     $this->output->outputLine('%s Errors occurred during child node creation', array($nodeCreationExceptions));
                 }
+                $this->persistenceManager->persistAll();
             } else {
-                $this->output->outputLine('%s missing child nodes need to be created', array($createdNodesCount));
+                if ($createdNodesCount > 0) {
+                    $this->output->outputLine('%s missing child nodes need to be created', array($createdNodesCount));
+                }
+                if ($updatedNodesCount >0) {
```
Comment (johannessteu) [inline TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Command/NodeCommandControllerPlugin.php]: missing space

**Comment by kdambekalns** [comment]
Nice one. _Could_ go into 2.1 but that's just optional. :)

**Comment by kitsunet** [comment]
I am all for adding this into 2.1. It's probably helpful to have in there and I don't see any harm done.

**Comment by robertlemke** [comment]
I'll push this again for 2.1

**Pull request closed without merging**